dueutil/game/helpers/shopabstract.py

```
@@ -103,3 +103,21 @@ def item_equipped_on_buy(self, player, item_name):
+
+    item_type = "global weapon"
+    inventory_slot = "global_weapons"
+
+    def item_equipped_on_buy(self, player, item_name):
+        if player.equipped["weapon"] == "none":
+            player.weapon = item_name
+            return True
+        return False
+
+    def get_item(self, item_name):
+        return weapons.get_weapon_for_server("global", item_name)
```

Ruff

Undefined name `weapons`

(F821)